File: CeptSportApp/app/src/main/java/ru/sport/ceptsportapp/StartActivity.java
package ru.sport.ceptsportapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.constraintlayout.widget.ConstraintLayout;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.graphics.Bitmap;
import android.graphics.drawable.BitmapDrawable;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.RelativeLayout;

import com.squareup.picasso.Picasso;
import com.squareup.picasso.Target;

public class StartActivity extends AppCompatActivity {
    private Button startButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_start);
        @SuppressLint({"MissingInflatedId", "LocalSuppress"}) RelativeLayout background = findViewById(R.id.start_back);
        String imageUrl = "http://159.69.90.204/api/CeptSportApp/background.jpg";
        Picasso.get().load(imageUrl).into(new Target() {
            @Override
            public void onBitmapLoaded(Bitmap bitmap, Picasso.LoadedFrom from) {
                background.setBackground(new BitmapDrawable(getResources(), bitmap));
            }

            @Override
            public void onBitmapFailed(Exception e, Drawable errorDrawable) {

            }

            @Override
            public void onPrepareLoad(Drawable placeHolderDrawable) {

            }
        });
        startButton = findViewById(R.id.start_button);
        startButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(StartActivity.this, QuestionActivity.class));
            }
        });
    }
}

File: KTmvvmQuize/app/src/main/java/com/sport/ktmvvmquize/StartActivity.kt
package com.sport.ktmvvmquize

import android.annotation.SuppressLint
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.LinearLayout

class StartActivity : AppCompatActivity() {
    @SuppressLint("MissingInflatedId")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_start)
        // Находим кнопку по id
        val startButton = findViewById<Button>(R.id.startButton)
        val imageUrl = "http://159.69.90.204/api/KTmvvmQuize/background.jpg"
        val linearLayout = findViewById<LinearLayout>(R.id.start_back)
        DownloadImageTask(linearLayout).execute(imageUrl)
        // Устанавливаем обработчик нажатия
        startButton.setOnClickListener {
            // Запускаем MainActivity
            val intent = Intent(this, MainActivity::class.java)
            startActivity(intent)

        }
    }
}